main.py

```diff
@@ -80,16 +82,21 @@ def execute_script(script_path):
     except:
         pass
 
-import comfy.utils
 import yaml
+from collections.abc import Callable
```

Comment:
Shouldn't `Callable` be imported from `typing` here?

Reply:
According to the Python documentation: 

> typing.Callable
    Deprecated alias to [collections.abc.Callable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable).
Deprecated since version 3.9: [collections.abc.Callable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).

Reply:
Thanks for explaining!